audbcards/core/templates/datacard.j2

```diff
@@ -0,0 +1,5 @@
+{% include 'datacard_header.j2' %}
+{% include 'datacard_description.j2' %}
+{% include 'datacard_example.j2' %}
+{% include 'datacard_tables.j2' %}
+{% include 'datacard_schemes.j2' %}
```

Comment:
Cool, this looks quite modular, so we can also easily extend it later on if another section is needed. Correct?